src/BootApp.ts

```diff
@@ -105,6 +106,14 @@ export class BootApp {
         }
     }
 
+    public get activeProfiles(): string | undefined {
+        return this._activeProfiles;
+    }
+
+    public set activeProfiles(value: string | undefined) {
+        this._activeProfiles = value || 'default';
```

Comment:
I would rather keep it blank than show a `profiles: 'default'` label when running with the default config.

Reply:
Sounds fair. Now with new commits, it's shouldn't display.
![default(no)_profile_from_prompt](https://github.com/microsoft/vscode-spring-boot-dashboard/assets/12153250/b8cdc85a-6e32-4b84-906c-40f4915acb53)